docs/mesos/README.md

```diff
@@ -0,0 +1,27 @@
+# Mesos with Calico Networking
+Calico provides networking for your Mesos cluster. These guides will get you started.
+
+## Preparing Hosts
+[This guide](PrepareHosts.md) will walk you through preparing each host in your cluster to be compatible with Calico and Mesos.
```

Comment:
I think SEO hates links that say "This Link" yadda yadda. I wrote these in when I was mocking out the link layout, but lets try to rephrase this sentence to have the guide name be the link. Something like:
```
The [Host Preperation Guide] will walk you through the necessary preparations on each host to get your Masters and Agents ready for Calico + Mesos.
```

Similar (but different wording) for the Prepare External Services guide as well.